frontend/app/tests/unit/specs/data/subscript-number.spec.ts

```diff
@@ -0,0 +1,196 @@
+import { mount } from '@vue/test-utils';
+import { createPinia, setActivePinia } from 'pinia';
+import { beforeEach, describe, expect, it } from 'vitest';
+import { BigNumber, bigNumberify } from '@rotki/common';
+import AmountDisplay from '@/components/display/amount/AmountDisplay.vue';
+import { useFrontendSettingsStore } from '@/store/settings/frontend';
+import { useGeneralSettingsStore } from '@/store/settings/general';
+
+describe('amountDisplay subscript', () => {
+  let pinia: ReturnType<typeof createPinia>;
+
+  beforeEach(() => {
+    pinia = createPinia();
+    setActivePinia(pinia);
+    const frontendStore = useFrontendSettingsStore();
+    frontendStore.update({
+      ...frontendStore.settings,
+      subscriptDecimals: true,
+      thousandSeparator: ',',
+      decimalSeparator: '.',
+      abbreviateNumber: false,
+      amountRoundingMode: BigNumber.ROUND_HALF_UP,
+      valueRoundingMode: BigNumber.ROUND_HALF_UP,
+    });
+  });
+
+  it('shows correct subscript count for small numbers', () => {
+    const generalStore = useGeneralSettingsStore();
+    generalStore.update({
+      ...generalStore.settings,
+      uiFloatingPrecision: 10,
+    });
+
+    const wrapper = mount(AmountDisplay, {
+      props: {
+        value: bigNumberify('0.0000000815'),
+        noScramble: true,
+        noTruncate: true,
+        fiatCurrency: null,
+        showCurrency: 'none',
+        forceCurrency: true,
```

Comment:
I have been trying for a while but it seems that without some of these options the tests fail. 

The problem is that these are mostly non-default options, the simplest case will be the one where you only supply the value. But from what I can see it fails. We need to figure out why, the test should be able to pass with only passing value to the wrapper.

It might be that there is some promise or something that needs to complete so things calling `await flushPromises() and `await nextTick()` might be required after mounting the component.